### ISSUE TITLE
STEM: Accessibility update to form elements

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#6291]
## Description
UPDATE: After reviewing the PR https://github.com/department-of-veterans-affairs/va.gov-team/issues/1874 and trying out the updated code, Trevor Pierce (I) have asked that the SR-only list items be removed from the `<legend>` of the Yes | No radio group. I've done a fair bit more testing since I wrote that ticket, and feel my initial worry about assistive tech users missing the list of reasons was unfounded.

The following feedback was provided by Trevor Pierce to the UX team regarding the updated to the 22-1995 as it relates to the STEM Scholarship updates: 

> I've looked through the wizard with an eye on the third radio, for the STEM scholarship. The only thing I can call out for accessible design is a consideration only, not a must do. Screen reader users sometimes move through forms using the | f | key to move by form elements. The text of requirements is not a form element, so there could be an opportunity for that to be missed by screen readers.
> 
> 
>~~ I can see a couple options that might work:~~
> 
> ~~1. Add a link that says something like "Review eligibility requirements" at the end of the "Based on the eligibility requirements above, do you want to apply for this scholarship?" <legend> text. This link would be an anchor that points to the paragraph above.~~
> ~~2. Adding an aria-describedby attribute to the <legend> that references a unique ID on the paragraph of requirements. This would need tested with screen readers, but either could give full context, if somewhat redundant.~~
> 
> ~~I'd rather come down on the side of redundancy than have important content get missed.~~

## Expected Results: 
1. Screen reader user are able to navigate the text of the STEM Scholarship Requirements using the | f | key

**Comment by ksantiagomorgan6** [comment]
pull request: 

https://github.com/department-of-veterans-affairs/vets-website/pull/11821

**Comment by sonntag-adam** [comment]
Testing passes.

**Comment by sonntag-adam** [comment]
Closing bug.
